product_docs/docs/transporter/ba_preview/getting_started/create_migration.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Creating a migration"
+---
+
+After you use EDB Transporter to read the source database, create a new migration in the EDB Postgres® AI Console. 
+This establishes a sync between the source database and a target cluster in the EDB Postgres AI Console. 
+
+1.  Access the [EDB Postgres AI Console](https://portal.biganimal.com) and log in with your EDB Postgres AI Database Cloud Service credentials.
+
+1.  Select the project where you created the database cluster. 
+
+1.  Within your project, select **Migrate** > **Migrations**. 
+
+1.  In the **Migrations** page, select **Create New Migration** > **To Managed Postgres**.
+
+1.  In the **Create Migration** page, assign a **Name** to the migration. 
+
+1.  Select the **Source** of the migration. The ID for the `cdcreader` is listed in the drop-down menu. 
+
+1.  Under **Destination**, select a target cluster for the migration and enter the name of the database where you want the migration to copy data and select **Next**. 
```

Comment:
Is the name of the user that created the target database important for the sync to work? Or can databases of any user be pushed? 

CC: @djw-m, since you brought this up.

Reply:
We handle the migration target-writing user completely internally, in the BigAnimal / PG AI Data Plane.

Reply:
This means that the customer does not have to do anything role/permissions-related, other than what we already list as part of the database preparations steps?
- [Creating new roles and grant access for CDC migration](https://deploy-preview-5759--edb-docs-staging.netlify.app/docs/transporter/latest/getting_started/preparing_db/preparing_postgres_source_databases/#create-new-roles-and-grant-acccess-for-cdc-migration) for postgres databases.
- [Create a user with limited privileges for data migration](https://deploy-preview-5759--edb-docs-staging.netlify.app/docs/transporter/latest/getting_started/preparing_db/preparing_oracle_source_databases/#create-a-user-with-limited-privileges-for-data-migration) for Oracle databases.